sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/QueueDescriptionExtensions.cs

```diff
@@ -94,50 +101,56 @@ private static QueueDescription ParseFromEntryElement(XElement xEntry)
-                        qd.LockDuration = XmlConvert.ToTimeSpan(element.Value);
-                        break;
-                    case "DefaultMessageTimeToLive":
-                        qd.DefaultMessageTimeToLive = XmlConvert.ToTimeSpan(element.Value);
-                        break;
                     case "MaxDeliveryCount":
                         qd.MaxDeliveryCount = Int32.Parse(element.Value);
                         break;
                     case "EnableBatchedOperations":
                         qd.EnableBatchedOperations = Boolean.Parse(element.Value);
                         break;
-                    case "Status":
-                        qd.Status = (EntityStatus)Enum.Parse(typeof(EntityStatus), element.Value);
-                        break;
-                    case "AutoDeleteOnIdle":
-                        qd.AutoDeleteOnIdle = XmlConvert.ToTimeSpan(element.Value);
+                    case "AuthorizationRules":
+                        qd.AuthorizationRules = AuthorizationRules.ParseFromXElement(element);
                         break;
-                    case "EnablePartitioning":
-                        qd.EnablePartitioning = bool.Parse(element.Value);
+                    case "IsAnonymousAccessible":
+                        qd.IsAnonymousAccessible = Boolean.Parse(element.Value);
                         break;
-                    case "UserMetadata":
-                        qd.UserMetadata = element.Value;
+                    case "Status":
+                        qd.Status = (EntityStatus)Enum.Parse(typeof(EntityStatus), element.Value);
                         break;
                     case "ForwardTo":
                         if (!string.IsNullOrWhiteSpace(element.Value))
                         {
                             qd.ForwardTo = element.Value;
                         }
                         break;
+                    case "UserMetadata":
+                        qd.UserMetadata = element.Value;
+                        break;
+                    case "SupportOrdering":
+                        qd.SupportOrdering = Boolean.Parse(element.Value);
+                        break;
+                    case "AutoDeleteOnIdle":
+                        qd.AutoDeleteOnIdle = XmlConvert.ToTimeSpan(element.Value);
+                        break;
+                    case "EnablePartitioning":
+                        qd.EnablePartitioning = bool.Parse(element.Value);
+                        break;
                     case "ForwardDeadLetteredMessagesTo":
                         if (!string.IsNullOrWhiteSpace(element.Value))
                         {
                             qd.ForwardDeadLetteredMessagesTo = element.Value;
                         }
                         break;
-                    case "AuthorizationRules":
-                        qd.AuthorizationRules = AuthorizationRules.ParseFromXElement(element);
+                    case "EnableExpress":
```

Comment:
Do we need to parse EntityAvailabilityStatus?

Reply:
No. That is entity runtime information.

---
In reply to: [516222109](https://github.com/Azure/azure-sdk-for-net/pull/16488#discussion_r516222109) [](ancestors = 516222109)